app/controllers/project/index.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember';
+
+const { Controller, computed, inject: { service } } = Ember;
```

Comment:
Think we could break this onto multiple lines? I find the `inject: { service }` hard to read when it is on one line with everything else.